ambry-replication/src/main/java/com.github.ambry.replication/ReplicaThread.java

```diff
@@ -721,6 +701,10 @@ private void writeMessagesToLocalStoreAndAdvanceTokens(List<ExchangeMetadataResp
                     + remoteReplicaInfo.getReplicaId(), e);
               }
             }
+          } else if (partitionResponseInfo.getErrorCode() == ServerErrorCode.Blob_Deleted) {
+            replicationMetrics.blobDeletedOnGetCount.inc();
+            logger.trace("One of the blobs to GET is deleted: Remote node: {} Thread name: {} Remote replica: {}",
```

Comment:
Ah, the error code isn't required in this statement anymore. Didn't delete it after the copy-paste. Removed it now, thanks !